apps/simplex-chat/Simplex/Chat.hs

```diff
@@ -145,12 +148,19 @@ chatSubscriber = do
                   setActive $ ActiveC c
                 _ -> pure ()
             _ -> pure ()
-      AgentTransmission {agentConnId, chatDirection = ReceivedDirectMessage Contact {localContactRef = c}, agentMessage} ->
+      AgentTransmission {agentConnId, chatDirection = ReceivedDirectMessage NewContact {activeConn}, agentMessage} ->
+        void . runExceptT $ case agentMessage of
+          CONF confId connInfo -> do
+            -- TODO update connection status
+            saveContact userId activeConn connInfo
```

Comment:
this could be done in separate scope, but maybe add primitive user dialogue asking to accept counterparty by profile?

Reply:
yes, possibly later, after protocol is stabilised. Also atm we don't have UX support for such interactions, it would conflict with how we process input as it.